### PR TITLE
Add support for using stretchr/testify during tests

### PR DESCRIPTION
**Pull request by gaby**
- Use https://github.com/stretchr/testify in place of `utils.AssertEqual()` for all storage drivers
- Update `Redis Cluster` GitHub Action
- Cleanup all `go.mod` files
- Update min version of all drivers to `go1.19`
- Added `go1.21` to all test files
- Migrate all usage of gofiber/utils to `v2`

### CODE REVIEW

**Comment by gaby** [comment]
Should the major version of the drivers that no longer support `go1.17` and `go1.18` be bump?

File: .github/workflows/test-azureblob.yml [hunk 1]
```diff
@@ -15,9 +15,9 @@ jobs:
         strategy:
             matrix:
                 go-version:
-                    - 1.18.x
```
Comment (ReneWerner87) [inline .github/workflows/test-azureblob.yml:18]: don´t forget to update the readme
image

**Review by ReneWerner87 — CHANGES_REQUESTED** [review]
almost perfect, just one little adjustment needed

File: .github/workflows/test-azureblob.yml (hunk 1 above)
Comment (gaby) [reply .github/workflows/test-azureblob.yml:18]: Oops! Will do

File: azureblob/azureblob_test.go [hunk 2]
```diff
@@ -88,50 +90,48 @@ func Test_AzureBlob_Get_NotExist(t *testing.T) {
...
-	utils.AssertEqual(t, nil, err)
+	require.Nil(t, err)
 
 	err = testStore.Set("john2", val, 0)
-	utils.AssertEqual(t, nil, err)
+	require.Nil(t, err)
 
 	err = testStore.Reset()
-	utils.AssertEqual(t, nil, err)
+	require.Nil(t, err)
```
Comment (efectn) [inline azureblob/azureblob_test.go:108]: require.NoError would be better

**Comment by gaby** [comment]
> Should the major version of the drivers that no longer support `go1.17` and `go1.18` be bump?

^Only thing still `TODO`, is this.

**Comment by ReneWerner87** [comment]
> > Should the major version of the drivers that no longer support `go1.17` and `go1.18` be bump?
> 
> ^Only thing still `TODO`, is this.

yes lets do this

**Review by ReneWerner87 — APPROVED** [review]

**Comment by gaby** [comment]
> > > Should the major version of the drivers that no longer support `go1.17` and `go1.18` be bump?
> > 
> > 
> > ^Only thing still `TODO`, is this.
> 
> yes lets do this

Done, most of the drivers are now `v2` except for like 2-3 that are now `v3`. I also updated all the README files.

**Review by ReneWerner87 — APPROVED** [review]

**Pull request merged**